index.css

```diff
@@ -847,34 +848,119 @@ button :disabled {
   background: #f4f1f0;
 }
 
-.other_works_item :hover + .action {
+.required::before {
+  content: "*";
+  color: red;
+  float: right;
+}
+
+/* The Modal (background) */
+#myModal {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  padding-top: 100px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+/* Modal Content */
+.modal_content {
+  background-color: #fefefe;
+  margin: auto;
+  padding: 20px;
+  border: 1px solid #888;
+  width: 80%;
+}
+
+/* The Close Button */
+.close {
+  color: #aaa;
+  float: right;
+  font-size: 28px;
+  font-weight: bold;
+}
+
+.modal_header h4 {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
```

Comment:
I've noticed that you added a `font-family` to all your elements. Although, your webpage is still displaying the default `font-family` 🤔. Please make sure to check if your `font-family` is correctly set up, to follow the original design
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/209199004-7a696095-8405-4009-af2c-25b18667d745.png)  | ![image](https://user-images.githubusercontent.com/42799579/209198972-5ddd9d87-acbb-46c8-97de-0037483b38e9.png)  |

It is very subtle, but please notice that the fonts are different. If you need any help, I hope I can help you:
To use custom fonts, you will need to first add them to your project. The easiest way to do it is by adding the font to your HTML `head`. I will explain to you a way of doing this using [Google Fonts](https://fonts.google.com/) 😄: 

- Search for the font that you want: 
![image](https://user-images.githubusercontent.com/42799579/198803592-001b8e4a-8f09-40bb-b094-d8196a4bcdab.png) 

- Select it: 
![image](https://user-images.githubusercontent.com/42799579/198803724-bb8ef580-55e5-454c-a351-25f326eb2992.png) 

- You will be taken to the font page, scroll down until you have the weights: 
![image](https://user-images.githubusercontent.com/42799579/198804057-63e483f4-fc4e-469b-b9ac-cf6d611be885.png) 

- Select the ones that you want by clicking here: 
![image](https://user-images.githubusercontent.com/42799579/198805635-c8e31d9e-82f6-4c72-8bdf-ffe170f5683f.png) 

- A box will appear on the right, after selecting all the fonts that you want, copy the link: 
![image](https://user-images.githubusercontent.com/42799579/198804518-6865e9ea-f5da-4636-9778-a7b3f6bc12a2.png) 

And add the copied link to your HTML `head` 😉. 
> Take into account that you can select multiple fonts & weights 😃